### ISSUE TITLE
Bug with control +z in windows

### ISSUE THREAD

**Issue by AlfredoArt** [synfig/synfig#355]
**Synfig 1.3.2      windows 10**:

at the time that i press control + z for correct my error this version closes unexpectedly

**Comment by morevnaproject** [comment]
Hello! Does this happens every time?

**Comment by AlfredoArt** [comment]
yes, it does. Each twice control z
thanks for your answer

**Comment by morevnaproject** [comment]
I cannot reproduce this. Can you please record a video showing how the problem happens?

**Comment by morevnaproject** [comment]
Ah, wait... I did tested wrong version. Let me try again...

**Comment by morevnaproject** [comment]
I am unable to reproduce this with Synfig 1.3.2 (32bit an 64bit). Can you please record a video of how this happens for you?

**Comment by ice0** [comment]
Can't reproduce it too. So i close this, until bug is showed again :)
